suite-native/module-accounts/src/components/AccountDetailHeader.tsx

```diff
@@ -46,7 +49,7 @@ export const AccountDetailHeader = memo(({ accountKey, accountName }: AccountDet
                 </Text>
             ) : (
                 <>
-                    <Graph points={points} loading={loading} />
+                    <Graph points={extendedPoints} loading={loading} />
```

Comment:
nit: enhancedPoints sounds better to me but I guess it's pretty much same so it's up to you

Reply:
renamed